src/motd.rs

```diff
@@ -62,6 +84,20 @@ impl Motd {
               minutes
             )
         });
+        // u64 max: 18446744073709551615(20 bytes length)
+        let result = self.counters_regex.replace(&result, || {
+            let tcp_incoming = self.counters.tcp.counters.incoming();
+            let tcp_outgoing = self.counters.tcp.counters.outgoing();
+            let udp_incoming = self.counters.udp.counters.incoming();
+            let udp_outgoing = self.counters.udp.counters.outgoing();
+            format!(
+                "Tcp: incoming {:>#20}, outgoing {:>#20}, Udp: incoming {:>#20}, outgoing {:>#20}",
```

Comment:
Also a possible improvement: instead of 1000 we can put 1k, instead of 1500000 we can put 1,5M.
@kurnevsky ?

Reply:
I think it is good idea.

Reply:
Right, it's for humans anyway.

Reply:
done